code/game/jobs/job/special_vr.dm

```diff
@@ -72,17 +72,17 @@
 	departments = list(DEPARTMENT_CIVILIAN)
 	department_flag = CIVILIAN
 	faction = "Station"
-	total_positions = -1
-	spawn_positions = -1
+	total_positions = 1
```

Comment:
Why make total positions for clown/mime limited too? IIRC -1 made them unlimited which allowed those whitelisted join even when there already were people around. Admittedly rare thing to have two people whitelisted for same job, but definitely a possibility.

Reply:
I didn't understand that's how that worked, and that's perfectly fine. I'll revert this.

Reply:
I've reverted this.